golang/cosmos/x/vstorage/vstorage_test.go

```diff
@@ -255,14 +255,13 @@ func TestSetWithoutNotify(t *testing.T) {
 
 // TODO: TestChildrenAndSize
 
-// TODO: TestValues
-
 func TestEntries(t *testing.T) {
 	kit := makeTestKit()
 	keeper, handler, ctx, cctx := kit.keeper, kit.handler, kit.ctx, kit.cctx
 
 	keeper.SetStorage(ctx, types.NewStorageEntry("key1", "value1"))
 	keeper.SetStorage(ctx, types.NewStorageEntry("key1.child1.grandchild1", "value1grandchild"))
+	keeper.SetStorage(ctx, types.NewStorageEntryWithNoData("key1.child1.grandchild2"))
```

Comment:
Curious why we need to test this case as it's not supposed to ever happen.

Reply:
Robust handling if that ever changes (or better yet, a failure in this test).